8976 - section 4 copy

brianpetway opened the pull request:
https://federal-spending-transparency.atlassian.net/browse/DA-8976

The download file and the 'as of' date for the download will be done with ticket 8979.

anjenkin reviewed (changes requested):
Text looks good but missing social share updates - 

Social Share Copy

#DYK the federal government is one of the largest and most consistent funding sources for Research & Development in the U.S.? Learn more by visiting #Data Lab’s analysis, R&D in Contract Funding! #OpenData https://datalab.usaspending.gov/rd-in-contracting/#section-studies

For the Email share option, there is a line break after “#OpenData” and the following displays below it: Check out this site https://datalab-usaspending.gov/rd-in-contracting/#section-studies

brianpetway commented:
I fixed the social share copy and the issues with line breaks in it. Also changed the link in footnote 5, that Grace just changed this morning. Ready to review again.
